app/controllers/profile_controller.rb

```diff
@@ -4,10 +4,13 @@ class ProfileController < ApplicationController
 
   def index
     cacher = Pii::Cacher.new(current_user, user_session)
-    @decrypted_pii = cacher.fetch
-    @recovery_code = flash[:recovery_code]
-    flash.delete(:recovery_code)
 
-    @has_password_reset_profile = current_user.password_reset_profile.present?
+    @view_model = UserProfile::ProfileIndex.new(
```

Comment:
we do have some other view models in this proj, might make sense to follow their naming convention